GitHubConfiguration.ps1

```diff
@@ -953,28 +969,27 @@ function Clear-GitHubAuthentication
         [switch] $SessionOnly
     )
 
-    Write-InvocationLog
+    if (-not $PSCmdlet.ShouldProcess('GitHub Authentication', 'Clear'))
+    {
+        return
+    }
 
     Set-TelemetryEvent -EventName Clear-GitHubAuthentication
```

Comment:
Just from a pattern consistency perspective, can you put this _after_ the `Write-InvocationLog` call?